### PR TITLE
[12.0][ADD] Email templates for closed tickets and changed state

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
See #66

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@miguel-domatix @alvaroDomatix @SamuelDomatix and @pedrobaeza Please, could yo check this PR

File: helpdesk_mgmt/i18n/helpdesk_mgmt.pot [hunk 1]
```diff
@@ -773,7 +773,21 @@ msgid "Teams"
 msgstr ""
 
 #. module: helpdesk_mgmt
+<<<<<<< HEAD
```
Comment (rvalyi) [inline helpdesk_mgmt/i18n/helpdesk_mgmt.pot]: you committed code with a conflict!

**Review by rvalyi — CHANGES_REQUESTED** [review]

File: helpdesk_mgmt/i18n/helpdesk_mgmt.pot (hunk 1 above)
Comment (emagdalenaC2i) [reply helpdesk_mgmt/i18n/helpdesk_mgmt.pot]: fixed

**Comment by emagdalenaC2i** [comment]
@miguel-domatix @alvaroDomatix @SamuelDomatix and @pedrobaeza Please, could yo check this PR

**Comment by emagdalenaC2i** [comment]
@miguel-domatix please, could you review and confirm that the migration of your PR is Ok.

@alvaroDomatix @SamuelDomatix and @pedrobaeza please review this PR.
Then I will check if there is any other PR pending to migrate to v12

**Review by miguel-domatix — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@rvalyi please update your review

**Comment by emagdalenaC2i** [comment]
@rvalyi please, could you update your review

**Review by rvalyi — APPROVED** [review]
Looks good now. Thanks!

**Pull request merged**